packages/client/webapp/src/components/Header.vue

```diff
@@ -1,33 +1,71 @@
 <script setup lang="ts">
-import { computed } from 'vue';
+import Button from '@/components/Button.vue';
 import IconLogin16 from '@carbon/icons-vue/es/login/16';
+import IconSearchLocate16 from '@carbon/icons-vue/es/search--locate/16';
+import { useRouter } from 'vue-router';
 import { useAuthStore } from '../stores/auth';
 
 const auth = useAuthStore();
-const isAuthenticated = computed(() => auth.isAuthenticated);
+const router = useRouter();
 
 const login = () => window.location.assign('http://localhost:8078');
+const homepage = () => router.push('/');
```

Comment:
Nitpick: I think it's a good convention to name functions/methods starting with a verb, e.g. `goToHomePage`